hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -478,6 +478,18 @@ public void run() {
     }
 
     @Override
+    public void reset() {
+        for (BasicInvocation invocation : invocations.values()) {
+            try {
+                invocation.notify(new MemberLeftException());
```

Comment:
Yes, we're already using that with the same purpose. And this method is here for a long time. It's not a new addition. Please review only changes in this PR.